### PR TITLE
SIE-105 added a reset user password method into user.js

### PR DESCRIPTION
**Pull request by dustinlo**

### CODE REVIEW

**Comment by dustinlo** [comment]
Modifying it since there are errors.

**Comment by dustinlo** [comment]
I thought this was a eslint problem but this is not. Can anyone help me out on this one? 
It's showing these errors when I perform npm start

image

**Review by quanganhhoang — APPROVED** [review]
AWESOME! Thank you so much for fixing my buggy code!

File: src/stories/pages/Signup/Signup.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ import { Button } from '../../components/Button/Button';
 import { Section } from '../../components/Section/Section';
 import { SplitGradient } from '../../layouts/SplitGradient/SplitGradient';
 import { Footer } from '../../layouts/Footer/Footer';
-import UserServices from '../../../firebase/CRUDServices/userServices';
+// import UserServices from '../../../firebase/CRUDServices/userServices';
```
Comment (quanganhhoang) [inline src/stories/pages/Signup/Signup.js:14]: let's delete these comments if the other stuff works

File: src/stories/pages/Signup/Signup.js [hunk 2]
```diff
@@ -43,7 +45,7 @@ export const Signup = ({ isDarkTheme }) => {
   const postUser = (e) => {
     e.preventDefault();
     // call user service
-    const service = UserServices.getInstance();
+    // const service = UserServices.getInstance();
```
Comment (quanganhhoang) [inline src/stories/pages/Signup/Signup.js:48]: same here

Comment (dustinlo) [reply src/stories/pages/Signup/Signup.js]: done

File: src/stories/pages/Signup/Signup.js (hunk 1 above)
Comment (dustinlo) [reply src/stories/pages/Signup/Signup.js]: done

**Review by theodore3131 — APPROVED** [review]

**Pull request merged**